android/preprocess/xml/menu_main_menu_xml.py

```diff
@@ -30,13 +39,8 @@ def _ProcessXML(root):
           '</menu> '\
           '</item>'
   vpn_node = ET.fromstring(vpn_node_str, parser=ET.XMLParser(encoding="utf-8"))
-  vpn_child = parent.find('item/[@android:id="@+id/update_menu_id"]', namespaces=ns)
+  vpn_child = parent.find('item/[@android:id="@+id/set_default_browser"]', namespaces=ns)
```

Comment:
Not sure I got the order, isn't vpn supposed to be below `Settings`?

Reply:
We decided to keep it after rewards menu as set as default browser takes higher priority.

Reply:
I see, it's good then.